redirects.txt

```diff
@@ -315,4 +315,7 @@ developer/reference/javascript/qweb.rst                          developer/refer
 developer/reference/backend/assets.rst                           developer/reference/frontend/assets.rst
 
 developer/reference/frontend/owl_component_system.rst            developer/reference/frontend/owl_components.rst
-developer/reference/frontend/generic_components.rst              developer/reference/frontend/owl_components.rst
\ No newline at end of file
+developer/reference/frontend/generic_components.rst              developer/reference/frontend/owl_components.rst
+
+# The url was already written in Odoo but has changed since the publication of this documentation 
+applications/general/product_images.rst                          applications/sales/sales/products_prices/products/product_images.rst
```

Comment:
```suggestion
applications/general/product_images.rst  applications/sales/sales/products_prices/products/product_images.rst  # fix a wrong target hard-coded in odoo
```
Also please make it the first redirect after "Redirections introduced in 15.0" to avoid FWP conflicts and somewhat follow the doc structure.